cico_build_deploy.sh

```diff
@@ -52,6 +52,11 @@ rm -rf ${TARGET_DIR}/
 docker build -t ${BUILDER_IMAGE} -f Dockerfile.build .
 
 mkdir ${TARGET_DIR}/
+mkdir ${TARGET_DIR}/images
+
+#This sets the permissions on target dir since this script may be invoked as root, but container is not.
+chmod -R 0777 ${TARGET_DIR}/
```

Comment:
This invocation needs to be after line 57 (the `build_guides.sh` invocation) because the files created by the build are then still owned by root and therefore impossible to delete by the regular user.